app/src/main/java/pl/bk20/forest/stats/presentation/StatsDetailsFragment.kt

```diff
@@ -52,7 +52,9 @@ class StatsDetailsFragment : Fragment() {
         )
         binding.apply {
             textStepCount.text = stepsText
-            viewGroupTree.isVisible = treeCollected
+            textTreeCollected.text = resources.getQuantityString(
+                R.plurals.trees_collected_format, treesCollected, treesCollected
+            )
```

Comment:
Create a new `val`, name it `treesCollectedText` and assign it this value. Then use it here. Also, change the id of the element that displays the text from textTreeCollected to textTreesCollected (plural form).